.github/workflows/build_and_tests.yml

```diff
@@ -11,6 +11,12 @@ on:
       - master
 
 jobs:
+  check_dependencies_compatibility:
+    runs-on: ubuntu-latest
+    - name: Check dependencies compatibility
+      run: |
+        pip install --extra-index-url https://test.pypi.org/simple/ -U .[tests,all_backends]
```

Comment:
You can add a `python -c "from ragger.backends import *` just to trigger the dependencies high-level imports and check all is nice.

Reply:
I've added the check for all top level modules